packages/common/src/core/HubItemEntity.ts

```diff
@@ -383,4 +389,32 @@ export abstract class HubItemEntity<T extends IHubItemEntity>
     );
     this.update({ typeKeywords, isDiscussable } as Partial<T>);
   }
+
+  /**
+   * Return a list of IAssociationInfo objects representing
+   * the associations this entity has, to the specified type
+   * @param type
+   * @returns
+   */
+  listAssociations(type: AssociationType): IAssociationInfo[] {
+    return listAssociations(this.entity, type);
```

Comment:
seems a little odd that the add and remove functions don't follow the signature pattern of list where the entity comes first.

Reply:
good point - when I was writing the tests I had thought about this, but likely got distracted and did not implement. Changed.